File: DayFour/Java/TwoSum.java
public class TwoSum{
    public static int[] getNumbers(int[] array, int number){
        int sumCount = 0;
        int[] result = new int[2];
        for(int index = 0; index < array.length; index++){
            for(int count = index +1; count < array.length; count++){
                if(array[index] + array[count] == number){
                    result[0] = array[index];
                    result[1] = array[count];
                    //return result;       
                }
               
            }
        }
        return result;
        
    }
    
    
}

File: DayFour/JavaScript/TwoSum.js

    function getNumbers(let array, let number){
        let sumCount = 0;
        let result = [];
        for(let index = 0; index < array.length; index++){
            for(let count = index +1; count < array.length; count++){
                if(array[index] + array[count] == number){
                    result[0] = array[index];
                    result[1] = array[count];
                    return result;       
                }
               
            }
        }
        return array;
        
    }
    
   
        int[] array = {2,5,6,3,-6};
        int number = 8;
        console.log(getNumbers(array, number));
    
    }
}
